people.qmd

```diff
@@ -29,6 +29,7 @@ JoVI also owes a huge debt to [Steve Haroz](http://steveharoz.com/), who was ins
 - Arvind Satyanarayan, MIT CSAIL
 - Carlos Scheidegger, Posit
 - Jon Schwabish, Urban Institute
+- Jeffrey Heer, University of Washington
```

Comment:
didn't we originally sort them alphabetically by last name, so Jeff should come after Jason Dykes? I'm a nitpicker, I know 😉